contributing/alpha_components.md

```diff
@@ -0,0 +1,25 @@
+# Alpha program for components
+
+The intent of the Alpha program is to provide a place for component code to land that may not be
+fully ready for production, but for which we still want active collaboration with the team and
+potentially some early adoption with select clients.
```

Comment:
Internally we define alpha as we want no clients. I'd like us to match to reduce confusion.

Reply:
Ah. This sounds like this matches out Beta program internally then. Is that acurate?